addons/mail/static/src/messaging/component/notification_list/notification_list.js

```diff
@@ -68,7 +69,7 @@ class NotificationList extends Component {
      */
     _useStoreSelector(props) {
         const threads = this._useStoreSelectorThreads(props);
-        const notifications = threads
+        let notifications = threads
```

Comment:
it has become quite misleading to name them "notifications" now that there's a `Notification` entity. I was looking fora relational field `notificationGroup` in `Notification`, just realising `storeProps.notifications` is not a list of `Notification` entities...

Reply:
Yes, this must be done as part of the clean up task-2258605